frame/multisig/src/lib.rs

```diff
@@ -164,49 +196,27 @@ decl_error! {
 		/// The data to be stored is already stored.
 		AlreadyStored,
 	}
-}
 
-decl_event! {
-	/// Events type.
-	pub enum Event<T> where
-		AccountId = <T as system::Config>::AccountId,
-		BlockNumber = <T as system::Config>::BlockNumber,
-		CallHash = [u8; 32]
-	{
+	#[pallet::event]
+	#[pallet::generate_deposit(pub(super) fn deposit_event)]
+	#[pallet::metadata(T::AccountId = "AccountId", T::BlockNumber = "BlockNumber")]
+	pub enum Event<T: Config> {
 		/// A new multisig operation has begun. \[approving, multisig, call_hash\]
-		NewMultisig(AccountId, AccountId, CallHash),
+		NewMultisig(T::AccountId, T::AccountId, CallHash),
 		/// A multisig operation has been approved by someone.
 		/// \[approving, timepoint, multisig, call_hash\]
-		MultisigApproval(AccountId, Timepoint<BlockNumber>, AccountId, CallHash),
+		MultisigApproval(T::AccountId, Timepoint<T::BlockNumber>, T::AccountId, CallHash),
 		/// A multisig operation has been executed. \[approving, timepoint, multisig, call_hash\]
-		MultisigExecuted(AccountId, Timepoint<BlockNumber>, AccountId, CallHash, DispatchResult),
+		MultisigExecuted(T::AccountId, Timepoint<T::BlockNumber>, T::AccountId, CallHash, DispatchResult),
```

Comment:
Does `Timepoint<T::BlockNumber>` correctly show up as `Timepoint<BlockNumber>` in the metadata when you use subsee?

Reply:
indeed, I modify the pallet::metadata it should be good now